x-pack/plugins/secops/public/components/page/network/ip_overview/field_renderers.test.tsx

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import AutonomousSystem = GetIpOverviewQuery.AutonomousSystem;
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+
+import { GetIpOverviewQuery, HostEcsFields, IpOverviewType } from '../../../../graphql/types';
+import { TestProviders } from '../../../../mock';
+
+import {
+  autonomousSystemRenderer,
+  dateRenderer,
+  hostNameRenderer,
+  locationRenderer,
+  whoisRenderer,
+} from './field_renderers';
+import { mockData } from './mock';
+
+describe('Field Renderers', () => {
+  describe('#locationRenderer', () => {
+    test('it renders correctly against snapshot', () => {
+      const wrapper = mount(
+        <TestProviders>
+          {locationRenderer(['source.geo.city_name', 'source.geo.region_name'], mockData.complete)}
+        </TestProviders>
+      );
+
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    test('it renders emptyTagValue when no fields provided', () => {
+      const wrapper = mount(
+        <TestProviders>{locationRenderer([], mockData.complete)}</TestProviders>
+      );
+      expect(wrapper.text()).toEqual('--');
```

Comment:
I recommend using:

```ts
expect(wrapper.text()).toEqual(getEmptyValue());
```
For all of these so if that value changes you won't ever be forced to update all of these tests.